Solobarine.github.io/portfolio.js

```diff
@@ -0,0 +1,202 @@
+const open = document.querySelector('#bars');
+const imgButton = document.querySelector('.img-button');
+const page = document.querySelector('.pop-up');
+const portfolio = document.querySelector('.the-nav1');
+const about = document.querySelector('.the-nav2');
+const contact = document.querySelector('.the-nav3');
+
+open.addEventListener('click', () => {
+  if (page.className === 'pop') {
+    page.className = '';
+  } else {
+    page.className = 'pop';
+  }
+});
+
+imgButton.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+portfolio.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+about.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+contact.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+// Popup Window
+const projectItems = {
+  title: 'Keeping track of hundreds of components website.',
+  skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
+  summary: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  liveButton: 'See Live',
+  sourceButton: 'See Source',
+};
```

Comment:
- [ ] Your projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads).
Please make sure that each card should have a corresponding object of that data. 
`arr = [{ },{ },{ },.....]`